manager_helpers/dask_manager_template.py

```diff
@@ -0,0 +1,106 @@
+"""
+Dask Distributed Manager Helper
+
+Conditions:
+- Dask Distributed and Dask Job Queue (dask_jobqueue in Conda/pip)
+- Manager running on the head node
+- SLURM manager
+
+For additional information about the Dask Job Queue, please visit this site:
+https://jobqueue.dask.org/en/latest/
+"""
+
+# Fractal Settings
+# Location of the Fractal Server you are connecting to
+FRACTAL_ADDRESS = "localhost:7777"
+
+# How many cores per node you want your jobs to have access to
```

Comment:
Maybe something with the connotations: How many cores per node (at most the number of cores on a node). Same for memory. "access" seems a bit different.